### PR TITLE
[dy] Only try to interpolate variables if they are in the query

### PR DESCRIPTION
**Pull request by dy46**
# Summary

For sql blocks, we try to interpolate the query even if there are no interpolated variables in the block. I added a regex to check if there is the `{{ df_X }}` syntax in the query before trying to interpolate variables.

I also removed the git sync from api triggers because it can potentially cause issues.

# Tests

tested locally

cc: @wangxiaoyou1993

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
What bug does it fix?

**Comment by dy46** [comment]
@wangxiaoyou1993 this one: https://mageai.slack.com/archives/C03HTTWFEKE/p1690478813984429

> I have a desire to run a custom Raw SQL block for one data source (Postgres) after another custom Raw SQL block with a datasource (trino) where there is no dependency on the outputs of the previous block. Mage is telling me that it can not run my second block due to being unable interpolate one output into the other...even though im not actually using the output of the previous block anywhere. Any idea on how to make this happy?

**Comment by dy46** [comment]
actually i think it would be better to check each variable since that could cover more cases.

**Comment by wangxiaoyou1993** [comment]
> actually i think it would be better to check each variable since that could cover more cases.

Yes, could you check in the for loop?

**Comment by dy46** [comment]
@wangxiaoyou1993 moved into the for loop

**Review by wangxiaoyou1993 — APPROVED** [review]

File: mage_ai/server/server.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ async def main(
         print('User authentication is enabled.')
         # We need to sleep for a few seconds after creating all the tables or else there
         # may be an error trying to create users.
-        sleep(3)
+        sleep(5)
```
Comment (wangxiaoyou1993) [inline mage_ai/server/server.py:259]: Is this an intentional change?

Comment (dy46) [reply mage_ai/server/server.py:259]: yeah i wanted to give it a little longer before creating the default roles and users

**Pull request merged**